_posts/2017-07-22-Why be a part of PyData.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  Why be a part of PyData
+date:   2017-07-22 
+categories: community pydata python
+---
+
+
+
+**Why should you be a part of PyData Organisation?**
+
+So, are you wondering, what is so special about PyData? What sets the PyData commumity apart?
+
+Here's a list ( not an exhaustive one and definitely not the only points! ) I as a PyData Delhi member I have made:
+
+* Why should I only support PyData? There are so many more communities out there! Why join in a PyData Conference or a Meetup? Well, simply because PyData conference proceeds go to NUMFocus, who in turn are the supporter of many open source projects like Julia, NumPy, Jupiter and more. 
+In short, we __give back__ to the community, and encourage financial as well as community support of open-source tools.
+
+* We have an amazing community! The PyData community, and the experiences I have had with them till now, have all been great and extremely uplifting. We have __good inter-community support__ as well as __ regular meetups__, where we discuss and solve the problems our community members face. We are a friendly and a diverse group focused on __individual as well as collective intellectual growth__ . We help out each other!
+
+* We have really __good speakers and sessions__ on the latest buzz in technology and the world of Python. There are __advanced topics and talks__, and as a member, I got to witness these speakers, introducing us newbies to awesome and mind boggling tech, and we all keep coming back for more- better conversations and learning at every step.
+
+There are a numerous other reasons, including how PyData helps me feel about giving back to my local community and supporting diverse initiatives in technical communities.
+Infact, being an organiser for PyData Delhi Conf'17 has given me a high and a rush like no other. I feel proud to be a part of such an active community and give back to the community by helping my peers and introducing them to open-source. The experience with my fellow Organising Team members has been amazing so far and we can't wait for September to arrive!
+So whatever your personal reasons maybe, I hope that listing down a few of mine have given you something to dwell upon. Go on, join a community near you and be a part of change and growth.
+Perhaps, it has motivated you to attend the upcoming PyData Delhi conference [PyData Delhi conference](https://pydata.org/delhi2017/) ! Ciao, and see you there folks!
+
+Signing off,
+
+__Zoha Hamid__
```

Comment:
Again, can we change this to something like:

> written by @zohamid 
on behalf of PyData Delhi & WWC Delhi Team